README.md

```diff
@@ -159,22 +162,21 @@ Deploy `FybrikApplication` in `default` namespace:
 ```bash
 kubectl apply -f fybrikapplication.yaml -n default
 ```
-3.  Check if `FybrikApplication` successfully deployed:
+3.  Run the following command to wait until the `FybrikApplication` is ready:
```

Comment:
I suggest that you be more specific. For instance: wait until the Status of the `FybrikApplication` is 'ready'

Reply:
Done thanks